### PR TITLE
Added a error manager app and the ability to list existing error codes.

### LINKED ISSUE #3: Ajouter une gestion des codes d'erreur sur l'API
### Créer une class `ErrorCode`:

- description : Text
- code : CharField
- url : Text

### Créer une ListView listant les erreurs avec une datatables

Les datatables nous permettent d'avoir un encart de recherche et des filtres par colonne déjà tout automatiser.

Exemple: [Repository de Saskatoon](https://github.com/tiagovaz/saskatoon/blob/master/saskatoon/harvest/templates/harvest/harvest/list.html)

---

Cette classe permettra de définir tout les codes d'erreurs utiliser dans l'API afin de produire une documentation utilisateur et développeur plus efficace.

### PR DESCRIPTION
**Pull request by PhilConnor**
| Q                             | R
| ----------------------------- | -------------------------------------------
| Correction de bugs ?          | non
| Nouvelle Fonctionnalité ?     | oui
| Tickets (_issues_) concernés  | Fix #3

**Description :**
-Added a error manager app and the ability to list existing error codes.
-/error_manager/error_codes to view the existing error codes
-jquery and datatable have been added to the project as well

### CODE REVIEW

File: src/api/urls.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+urlpatterns = [
```
Comment (RignonNoel) [inline src/api/urls.py]: This file is not relevant in this issue

File: src/error_manager/templates/base.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{% load static %}
+<!DOCTYPE html>
+
+<html>
+
+    <head>
+		<link rel="stylesheet" href="{% static 'DataTables-1.10.12/media/css/jquery.dataTables.css' %}">
```
Comment (RignonNoel) [inline src/error_manager/templates/base.html]: Error in this line, bad path:

This line is the good:
**DataTables/**DataTables-1.10.12/media/css/jquery.dataTables.css

**Comment by RignonNoel** [comment]
I've take a look and i need two correction before i accept this PR:

 - Good path on the static of datatables css
 - Remove the sqllite of the commit

**Note:** Next time try to make more commit. Here it's a commit of 653 files xD

**Comment by PhilConnor** [comment]
@RignonNoel I forced pushed a cleaner commit history. I need your feedback on this. I also fixed what you pointed out. Also, i don't  know why but when I pushed the new commit history, it closed the pull request automatically. I re-opened it manually.

**Comment by RignonNoel** [comment]
@PhilConnor It's good for me! Just try to make short commit title. As you can see on github, the title of the last commit is not entirely display here.

But it's ok for the moment :)

**Pull request merged**